packages/admin/admin/src/FinalForm.tsx

```diff
@@ -100,8 +100,7 @@ export function FinalForm<FormValues = AnyObject>(props: IProps<FormValues>) {
                                 error: new SubmitError("Form has Validation Errors", formRenderProps.errors),
                             };
                         }
-
-                        const submissionErrors = await formRenderProps.form.submit();
```

Comment:
`formRenderProps.form.submit();` doesn't actually return the submissionErrors and it doesn't wait for the submission to be finished before returning